mobile.md

```diff
@@ -0,0 +1,35 @@
+# Mobile
+
+## Termin wykonania zadania:
+<!-- Dodaj termin wykonania zadania -->
+
+## Opis projektu
+Aplikacja mobilna wyświetlająca listę drinków wraz ze składnikami, zdjęciami oraz innymi szczegółami. Umożliwia przeglądanie drinków, ich filtrowanie oraz sortowanie.
+
+## API
+Do obsługi drinków wykorzystywane jest API bazujące na serwisie [TheCocktailDb](https://www.thecocktaildb.com/).
+- Dokumentacja API dostępna [tutaj](https://cocktails.solvro.pl/).
+
+## MVP (Minimum Viable Product)
+- Wyświetlanie listy koktajli (zdjęcie, tytuł).
+- Wyświetlanie szczegółów wybranego drinka (składniki i inne dane).
+
+## Wymagania
+- Aplikacja musi korzystać z zewnętrznego [API](https://cocktails.solvro.pl/).
+- Kod aplikacji musi być czytelny i dobrze podzielony na komponenty.
+- Responsywność i dostosowanie do różnych urządzeń.
+- Aplikacja powinna być estetyczna, intuicyjna i łatwa w obsłudze.
+- Dołącz dobrze napisane README z opisem technologii, funkcji aplikacji oraz screenshotami.
+
+## Technologie
+- **Flutter** (rekomendowane)
+- **Kotlin**, **Swift**, **Kotlin Multiplatform** (alternatywy)
+
+## Nice to have (opcjonalne funkcje)
+- Filtrowanie koktajli.
+- Sortowanie wyników.
+- Paginacja lub infinite scroll.
+- Wsparcie dla różnych języków (np. polski, angielski).
```

Comment:
nwm czy api to obsługuje więc można to zmienić na coś w stylu że Podstawy dla prostego wsparcia wielu języków